djantic/main.py

```diff
@@ -15,6 +17,10 @@
 from pydantic.errors import PydanticUserError
 from pydantic._internal._model_construction import ModelMetaclass
 
+if sys.version_info >= (3, 10):
+    from types import UnionType
+else:
+    from typing import Union as UnionType
```

Comment:
@idan-david This looks ok to me. The import on line 6 needs to be removed though. Let's see if the tests pass after this.

Reply:
Sorry for the mess, removed the import on line 6